packages/expo/src/Linking/LinkingModule.web.ts

```diff
@@ -0,0 +1,40 @@
+import NativeEventEmitter from 'react-native/Libraries/EventEmitter/NativeEventEmitter';
+import invariant from 'invariant';
+
+class RNLinking extends NativeEventEmitter {
+
+    constructor() {
+      super({ addListener() {}, removeListeners() {} })
+    }
+
+    async canOpenURL(url: string): Promise<boolean> {
+      this._validateURL(url);
+      return true;
+    }
+    async getInitialURL(): Promise<string> {
+      return window.location.href;
+    }
+    async openURL(url: string): Promise<void> {
+      this._validateURL(url);
+      window.location.href = new URL(url, window.location.href).toString();
```

Comment:
```suggestion
       window.location.href = url;
```

This should work and doesn't rely on the new `URL` API in browsers